.eslintrc.js

```diff
@@ -44,7 +52,7 @@ module.exports = {
                 'react/require-default-props': 'off',
                 'valid-jsdoc': 'off',
                 'es/no-optional-chaining': 'off',
-                'es/no-nullish-coalescing': 'off',
+                'es/no-nullish-coalescing-operators': 'off',
```

Comment:
What's the difference between the two ones?

Reply:
`es/no-nullish-coalescing` doesn't exist, I added it by mistake.